supabase/tests/billing.test.sql

```diff
@@ -159,8 +159,12 @@ begin
     catalog_name, grain, ts, flow_document, bytes_written_by_me, bytes_read_by_me, usage_seconds
   ) values
     ('aliceCo/aa/hello', 'monthly', '2022-08-01T00:00:00Z', '{}', 5.125 * 1024 * 1024 * 1024, 0, 3600 * 720),
```

Comment:
nit: these monthly grains can be removed altogether, right?

Reply:
Yep! Removed